pytorch_lightning/callbacks/base.py

```diff
@@ -98,7 +98,9 @@ def on_train_epoch_start(self, trainer: 'pl.Trainer', pl_module: 'pl.LightningMo
         """Called when the train epoch begins."""
         pass
 
-    def on_train_epoch_end(self, trainer: 'pl.Trainer', pl_module: 'pl.LightningModule', outputs: EPOCH_OUTPUT) -> None:
+    def on_train_epoch_end(
+        self, trainer: 'pl.Trainer', pl_module: 'pl.LightningModule', unused: Optional = None
```

Comment:
Stupid question, but can you explain why we need `unused` here?

Reply:
We still need a positional argument in case the callback continues to provide its own override with `outputs`
I was following @carmocca 's pattern from #6136 as a template

Reply:
Makes sense, thanks :smiley: